src/GameView.cpp

```diff
@@ -67,6 +102,31 @@ void GameView::update(sf::Event& Event, float dt){
     childState = new GameOver(*App, "You Lose...");
   }
 
+  if(levelManager.currentLevelName == "BEDROOM"){ //plan to change
+    sprite_holywater.setTexture(texture_water);
+    HolyWater water = this->room.getWaters().at(0);
+    sprite_holywater.setPosition(this->room.getWaters().at(0).position.x,this->room.getWaters().at(0).position.y);
+    for(int i = 0; i < this->room.getWaters().size(); i++){
```

Comment:
What I would do instead is first check if the player is pressing e, then if he is pressing e check if his coordinate matches with any of the items in the list

Reply:
lproblem with this current implementation is there are going to be more items than just water, so you have change the conditionals to all the interactable items (like the safe and notes) not just water